locales/pt-BR/translation.json

```
@@ -0,0 +1,93 @@
+  },
+  "$Preferences": {
+    "title": "Time to Leave - Preferências",
+    "userPreferences": "Preferences de usuário",
+    "workingDays": "Dias uteis",
+    "sun": "Dom",
+    "mon": "Seg",
+    "tue": "Ter",
+    "wed": "Qua",
+    "thu": "Qui",
+    "fri": "Sex",
+    "sat": "Sab",
+    "hideNonWorkingDay": "Ocultar finais de semana (Visão de mês)",
+    "hoursPerDay": "Horas por dia",
+    "appBehavior": "Comportamento do App",
+    "notification": "Notificação",
+    "allowRecurringNotifications": "Permitir notificações recorrentes",
+    "minutesBetweenNotifications": "Minutos entre notificações",
+    "startOnLogin": "Começar no Login",
+    "countTodayOnTotals": "Contar Hoje nos totais",
+    "overallBalanceStart": "Data de início do balanço total",
+    "minimizeToTray": "Botão de minimizar deve minimizar para o Tray",
+    "closeMinimizeToTray": "Botão de fechar deve minimizar para o Tray",
+    "themes": "Temas",
+    "systemDefault": "Padrão do Sistema",
+    "light": "Claro",
+    "dark": "Escuro",
+    "cadentStar": "Estrela Cadente",
+    "view": "Visão",
+    "month": "Mês",
+    "day": "Dia",
+    "numberOfEntries": "Número de entradas",
+    "fixed": "Fixo",
+    "flexible": "Flexível",
+    "language": "Idioma"
+  },
+  "$Menu": {
+    "about": "Sobre",
+    "all-clear": "Tudo limpo!",
+    "all-files": "Todos os Arquivos",
+    "cancel": "Cancelar",
+    "check-for-updates": "Checar por atualizações",
+    "clear-database": "Limpar base de dados",
+    "confirm-clear-all-data": "Tem certeza que quer limpar todos os dados?",
+    "confirm-import-db": "Tem certeza que deseja importar uma base de dados? Ela vai sobrescrever qualquer informação corrente.",
+    "confirm-migrate-to-flexible": "Tem certeza que deseja migrar da base de dados do calendário fixo para a do flexível?\n\nA base de dados do calendário flexível existente vai ser limpa.",
+    "copy": "Copiar",
+    "could-not-be-loaded": "não foram carregadas",
+    "cut": "Cortar",
+    "database-export": "Exportação de base de dados",
+    "database-imported": "Base de dados importada",
+    "database-migrated": "Banco de dados migrado",
+    "database-was-exported": "Ok, a base de dados foi exportada.",
+    "edit": "Editar",
+    "exit": "Sair",
+    "export-database": "Exportar base de dados",
+    "export-db-to-file": "Exportar Base de Dados para Arquivo",
+    "export": "Exportar",
+    "failed-entries": "Entradas falharam",
+    "failed-migrating": "Migração falhou",
+    "help": "Ajuda",
+    "import-database": "Importar base de dados",
+    "import-db-from-file": "Importar Base de Dados de um Arquivo",
+    "import-successful": "Uhul! Sucesso ao importar!",
+    "import": "Importar",
+    "menu": "Menu",
+    "migrate-calendar-to-flexible": "Migrar base de dados de calendário fixo para flexível",
+    "migrate-to-flexible-calendar": "Migrar para calendário flexível",
+    "migration-successful": "Uhul! Sucesso na migração!",
+    "no-thanks": "Não, obrigado",
```

```suggestion
    "no-thanks": "Não, valeu",
```
If you wanna go gender neutral (but a little bit more informal :D)

---

I like being informal 👍